src/doc/unstable-book/src/language-features/asm-experimental-arch.md

```diff
@@ -145,14 +151,19 @@ This feature tracks `asm!` and `global_asm!` support for the following architect
+| All          | `fr` (Hexagon), `fp` (PowerPC), `$fp` (MIPS), `Y` (AVR), `r4` (MSP430), `a6` (M68k), `r11` (s390x), `x29` (Arm64EC) | The frame pointer cannot be used as an input or output.                                                                                                                             |
 | All          | `r19` (Hexagon), `x19` (Arm64EC)        | This is used internally by LLVM as a "base pointer" for functions with complex stack frames.                                                                                        |
 | MIPS         | `$0` or `$zero`                         | This is a constant zero register which can't be modified.                                                                                                                           |
 | MIPS         | `$1` or `$at`                           | Reserved for assembler.                                                                                                                                                             |
 | MIPS         | `$26`/`$k0`, `$27`/`$k1`                | OS-reserved registers.                                                                                                                                                              |
 | MIPS         | `$28`/`$gp`                             | Global pointer cannot be used as inputs or outputs.                                                                                                                                 |
 | MIPS         | `$ra`                                   | Return address cannot be used as inputs or outputs.                                                                                                                                 |
 | Hexagon      | `lr`                                    | This is the link register which cannot be used as an input or output.                                                                                                               |
+| PowerPC      | `$r2`, `$r13`                           | These are system reserved registers.                                                                                                                                                |
+| PowerPC      | `$r29`, `$r30`                          | These are used internally by LLVM.                                                                                                                                                  |
+| PowerPC      | `lr`                                    | The link register cannot be used as an input or output.                                                                                                                             |
```

Comment:
I think `xer` should be not supported either and should occupy a line after `lr`.

Reply:
Currently, `xer` is supported as a clobber-only register.

https://github.com/rust-lang/rust/blob/d30a53e46962b7bd6bcc74e23cfb81c36e996eed/src/doc/unstable-book/src/language-features/asm-experimental-arch.md?plain=1#L39

See https://rust-lang.zulipchat.com/#narrow/stream/216763-project-inline-asm/topic/preserve_flags.20on.20OpenPower.2FPowerPC.3F/near/251292056 for more context.

Reply:
Cool, I thought these lines are for input or output operands. It makes sense `xer` as clobber operand since there are instructions changing the `ca` bit of `xer`.